recipe/meta.yaml

```
@@ -21,8 +21,8 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython <3                              # [build_platform != target_platform]
-    - sysroot_{{ target_platform }} 2.17     # [linux and cdt_name == "cos7"]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
```

nit: Let's match quotes here. Also slight preference for adding `stdlib` first

```suggestion
    - {{ stdlib("c") }}
    - {{ compiler("c") }}
```